src/client/pages/RegistrationEmailSentPage.tsx

```diff
@@ -12,8 +13,8 @@ export const RegistrationEmailSentPage = () => {
     <EmailSent
       email={email}
       subTitle={"We've sent you an email"}
-      previousPage={'/signin'}
-      resendEmailAction={'/register/email-sent/resend'}
+      previousPage={`${Routes.SIGN_IN}`}
+      resendEmailAction={`${Routes.REGISTRATION}${Routes.RESEND}`}
```

Comment:
Shouldn't this be `${Routes.REGISTRATION}${EMAIL_SENT}${Routes.RESEND}` ?

Reply:
there's a comment about this in the pr description

Reply:
👍